vr_client:
  vr_server_ip: "10.9.163.23"
  vr_server_port: 5555
  reconnect_interval: 5.0
  socket_timeout: 5.0